src/components/QuickGrid/rowGrouper.ts

```diff
@@ -5,19 +5,19 @@ import { groupBy } from '../../utilities/array';
 class RowGrouper {
     groupByColumns: Array<IGroupBy>;
     columns: Array<GridColumn>;
-    expandedRows: any;
+    collapsedRows: Array<string>;
 
-    constructor(groupByColumns, expandedRows, columns) {
+    constructor(groupByColumns, collapsedRows, columns) {
         this.groupByColumns = [...groupByColumns];
-        this.expandedRows = expandedRows;
+        this.collapsedRows = collapsedRows;
         this.columns = [...columns];
     }
 
-    isRowExpanded(columnName, name) {
+    isRowExpanded(name) {
         let isExpanded = true;
-        let expandedRowGroup = this.expandedRows[columnName];
-        if (expandedRowGroup && expandedRowGroup[name]) {
-            isExpanded = expandedRowGroup[name].isExpanded;
+        let index: number = this.collapsedRows.indexOf(name, 0);
```

Comment:
will this be a performance issue?

Reply:
Doesn't seem to be so at a data sample of 100k rows. Object (Map) would be faster, but working with an array is much cleaner, in terms of code readability. 
If objects were to be used, assignment would look like this: collapsedRows[groupKey] = {}, or collapsedRows[groupKey] = true,  but the value would never be used. Also removing properties from this structure (having 'true' or an empty object as value) wouldn't make sense to someone reading this code later, but keeping all the groupKeys as collapsed rows and assigning them boolean values doesn't either. Corrections and suggestions are welcome! :D